.github/workflows/canary-integration-test.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
     - name: test external script create-external-cluster-resources.py
       run: |
-        toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[0].metadata.name}')
+        toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
```

Comment:
If there is more than one pod, will only one be returned? That shouldn't happen with the toolbox, it's just a general question

Reply:
You will get all of them, but like you said this is safe since we only have one.